cli/src/commands/git/push.rs

```diff
@@ -766,15 +771,21 @@ fn find_bookmarks_targeted_by_revisions<'a>(
 ) -> Result<Vec<(&'a str, LocalAndRemoteRef<'a>)>, CommandError> {
     let mut revision_commit_ids = HashSet::new();
     if use_default_revset {
-        // remote_bookmarks(remote=<remote>)..@
-        let workspace_id = workspace_command.workspace_id();
-        let expression = RevsetExpression::remote_bookmarks(
-            StringPattern::everything(),
-            StringPattern::exact(remote_name),
+        let revs = workspace_command.settings().get_string("revsets.push")?;
+        // This is a complete hack, since we have no convenient API for this yet.
+        let mut aliases_map = RevsetAliasesMap::new();
+        aliases_map.insert("remote", remote_name).unwrap();
+        let extensions = RevsetExtensions::default();
+        let context = RevsetParseContext::new(
+            &aliases_map,
+            "TODO",
+            chrono::Utc::now().fixed_offset().into(),
+            &extensions,
             None,
-        )
-        .range(&RevsetExpression::working_copy(workspace_id.clone()))
-        .intersection(&RevsetExpression::bookmarks(StringPattern::everything()));
+        );
+        let mut diags = RevsetDiagnostics::default();
+        let (expression, _) = revset::parse_with_modifier(&mut diags, &revs, &context)?;
```

Comment:
This will probably help to set local variable `remote`. The first two patches are sent as #6142.

https://github.com/jj-vcs/jj/compare/main...yuja:jj:push-nuxxkpyxzprs